packages/eslint-plugin-next-on-pages/src/rules/no-unsupported-configs.ts

```diff
@@ -273,3 +277,11 @@ function extractModuleExportValue(node: Node): Node | null {
 
 	return null;
 }
+
+function extractESMExportValue(node: Node): Node | null {
```

Comment:
just for consistency with `extractModuleExportValue` could you adda tsdoc comment on top of the function? 🙂🙏

Reply:
Yes, of course, it's done.